src/main/java/myfirstproject/db/TemporalListOfOperations.java

```diff
@@ -0,0 +1,8 @@
+package myfirstproject.db;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TemporalListOfOperations {
```

Comment:
Why do you need this class?

Reply:
> 
I use this class for save all data from file which i read.

Reply:
You can just create a local variable